app/views/layouts/listnav/_cloud_object_store_container.html.haml

```diff
@@ -0,0 +1,35 @@
+- if @record.try(:name)
+  #accordion.panel-group
+    = miq_accordion_panel(truncate(@record.name, :length => truncate_length), true, "icon") do
+      = render(:partial  => "layouts/quadicon",
+               :locals   => {:mode => :icon,
+               :item     => @record,
+               :size     => 72,
+               :typ      => :listnav})
+
+    = miq_accordion_panel(_("Properties"), false, "cloud_object_store_container_prop") do
+      %ul.nav.nav-pills.nav-stacked
+        %li
+          = link_to(_('Summary'), {:action => 'show', :id => @record, :display => 'main'}, :title => _("Show Summary"))
+
+    = miq_accordion_panel(_("Relationships"), false, "cloud_object_store_container_rel") do
+      %ul.nav.nav-pills.nav-stacked
+        - if role_allows(:feature => "ems_cloud_show") && @record.ext_management_system
+          %li
+            = link_to("#{ui_lookup(:table => "ext_management_system")}: #{@record.ext_management_system.name}",
```

Comment:
Yes, if the inside of `link_to` is meant to be localized, then it needs to look the way you showed. And no,
at this moment, we don't have a gettext replacement for `ui_lookup()`, so it's still okay to use it.